client/hotrod-client/src/main/java/org/infinispan/client/hotrod/logging/Log.java

```diff
@@ -226,8 +226,8 @@
    IllegalStateException queryParameterNotSet(String filterConverterFactoryName);
 
    @LogMessage(level = WARN)
-   @Message(value = "Server not reachable whilst closing iteration '%s'", id = 4061)
-   void ignoringServerUnreachable(String iterationId);
+   @Message(value = "Ignoring error when closing whilst closing iteration '%s'", id = 4061)
```

Comment:
I've removed _whilst closing_ myself and pushed to master. Trying to rebase on 8.2.x...